### PR TITLE
[WIP] Add travis file and defaults

### PR DESCRIPTION
**Pull request by naxhh**
Not yet working but I will not be able to work on this until next week end.
So, if you want to take this over. be my guest :)

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+rabbitmq:
+    user: "user"
+    password: "password"
+    plugins: []
+
+vagrant_local:
```
Comment (naxhh) [inline defaults/main.yml:7]: This is a dependency between roles that should be fixed. But can be done in another pr

**Comment by debo** [comment]
@naxhh sure no problem. I'll have a look later today.

**Comment by naxhh** [comment]
If you don't have time I'll be working on this stuff from time to time. I've assigned you only for review purposes.

**Comment by debo** [comment]
Well it's midnight here so I'll probably go to sleep now, but I'll surely help you out.

**Comment by debo** [comment]
@naxhh all yours ;)

**Comment by naxhh** [comment]
Nice job!

**Pull request merged**